Automattic/jetpack
# Form Block: Add styling for input fields

This adds various style settings to the sidebar of the contact form input fields.

Styles such as:
* Corner Radius
* Border Width
* Line Height of input field
* Color of label text, input text, border and input field.

The attributes defined in the sidebar are copied to the PHP shortcode that creates the input fields when the form is displayed.

It's still very much a work in progress. 

Fixes #27525

#### Changes proposed in this Pull Request:

* Adds more sidebar controls for the input fields of the contact form.
* Reads those attributes and adds them to the shortcodes used to display the input fields.

#### Other information:


#### Jetpack product discussion
pcsBup-py-p2#comment-126

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Apply the patch
* Create a form block and click on the Name field.
* In the sidebar change the colours and other settings.
* Load/reload the page holding the form block to observe changes.
